core-customize/hybris/bin/custom/datahubtestsetup/resources/impex/essentialdata-datahubtestsetup.impex

```diff
@@ -1,10 +1,17 @@
 #% impex.setLocale( Locale.ENGLISH )
 
 # DataHub configurations
+INSERT_UPDATE DataHubUserModel    ;id[unique = true] ; userName                                          ; password
```

Comment:
userName and password attributes are not present in the version used by the sample. We need to release datahub-cs first. Then we need to change the commerce version in the manifest and only then we can add these impex statements.